getting-started/master-your-ui/the-table-view.md

```diff
@@ -36,6 +36,59 @@ The **extended search** lets you search also within _reference fields_ of `belon
 _Extended search_ is **not the default** search because it is **slower**.
 {% endhint %}
 
+### Advanced syntax for search
+
+{% hint style="info" %}
+_Advanced syntax for search_ is only supported on nodejs agents `@forestadmin/agent` starting from version 1.37. 
+{% endhint %}
+
+Search supports advanced syntax to help you find exactly what you are looking for:
+
+* `-term` will search for records that do not contain `term` ;
+* `property:term` will search for records that contain `term` in the `property` field ;
```

Comment:
Sadly the property expected is the technical one. I guess it is easier this way, but not very user friendly … 
<img width="1176" alt="image" src="https://github.com/ForestAdmin/documentation-user-guide/assets/19222576/064dae51-8543-4e19-9290-2d6cc965b405">